### DISCUSSION TITLE
github and x connections

### DISCUSSION THREAD

**Discussion by Carlos-Levi** [community/community#128910] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey guys, I hope all is well with you 

I created an account on 'x' (twitter) aimed at this dev 'bubble'. If you can and would like to follow, I will be very grateful :D

**If you want to connect here too, I will reciprocate everyone!** 

https://x.com/engcodee

**Comment by ebndev** [comment, marked as answer]
Hi @Carlos-Levi, thanks for sharing with the Community! 

Unfortunately, we currently do not allow self-promotion or advertising on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. As a result, I will be locking this post but hope you'll come back to visit and engage with users here!
